src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -570,4 +612,10 @@ private ColumnModel makeColumnModelFromColumn(Column column) {
             .datatype(column.getType())
             .arrayOf(column.isArrayOf());
     }
+
+    private static List<SnapshotRequestAccessIncludeModel> getDefaultIncludes() {
+        return Arrays.stream(StringUtils.split(SnapshotsApiController.RETRIEVE_INCLUDE_DEFAULT_VALUE, ','))
```

Comment:
This pains me.. if `RETRIEVE_INCLUDE_DEFAULT_VALUE` was an `EnumSet<>` then you could generate its current value using `String.join(",", RETRIEVE_INCLUDE_DEFAULT_VALUE)` and get rid of this code. You'd also have to change the method that accepts `List<SnapshotRequestAccessIncludeModel>` to accept `Collection<>` instead. (In general, I think most methods we have that accept `List` should instead accept `Collection`.)

Reply:
You and me both.  The problem is that it's used in an annotation and that doesn't let you perform operations.  You have to start from a value.  This seemed less bad than having yet one more place to copy the default value

Reply:
Yeah, confirming that for using the value in the defaultValue for the api controller it requires a constant

Reply:
Oh well.